_includes/apiSidebar.html

```diff
@@ -0,0 +1,42 @@
+<nav id="api" class="sidebar sticky">
+    <a href="{{ site.github.url }}/api/" class="item{% if page.url == '/api/index.html' %} active{% endif %}">
+        Introduction
+    </a>
+    <a href="{{ site.github.url }}/api/bdd/" class="item{% if page.url == '/api/bdd/' %} active{% endif %}">
+        Expect / Should
+    </a>
+    <a href="{{ site.github.url }}/api/assert/" class="item{% if page.url == '/api/assert/' %} active{% endif %}">
+        Assert
+    </a>
+    <a href="{{ site.github.url }}/api/plugins/" class="item{% if page.url == '/api/plugins/' %} active{% endif %}">
+        Plugin Utilities
+    </a>
+    <a href="{{ site.github.url }}/api/test/" class="item{% if page.url == '/api/test/' %} active{% endif %}">
+        Online Test Suite
+    </a>
+    {% if page.api_namespace %}
+    <div class="list box-wrap antiscroll-wrap" style="height: 338px;">
+        <div class="box">
+            <div class="antiscroll-inner" style="height: 338px; width: 225px;">
+                <div class="box-inner">
+                  {% for method in site.data.chai %}
+                    {% assign render = false %}
+                    {% assign name = false %}
+                    {% for tag in method.tags %}
+                      {% if tag.type == "namespace" and tag.string == page.api_namespace %}
+                        {% assign render = true %}
+                      {% endif %}
+                      {% if tag.type == "name" %}
+                        {% capture name %}{{ tag.string }}{% endcapture %}
```

Comment:
Maybe we could use `assign` instead of `capture` here, just to maintain the code uniform.
I also did some research on which is faster and [`assign` seems to be faster](https://github.com/Shopify/liquid/issues/487).

Anyway, that's just me being nitpicky 😆